src/main/java/frc/robot/RobotId.java

```diff
@@ -28,17 +34,83 @@ public enum RobotId {
         this.subsystems = List.of(subsystems);
     }
 
+    /**
+     * @Deprecated The RobotId enum should just be for robot names
+     * @return
+     */
+    @Deprecated
     public List<Class<? extends SubsystemBase>> getSubsystems() {
         return subsystems;
     }
```

Comment:
This isn't used anywhere; you can delete it.

Reply:
Generally, I'm reluctant to blow away code that may be used. That's why deprecated is useful.